COSC-499-W2023/year-long-project-team-1
# feat(api): modify video upload API to comply with new upload flow

# Welcome to PrivacyPal! 👋

Fixes: #361
Depends on #414 

## Description of the change:
Modifies the video upload API to upload files to S3 to trigger the processing lambda

## Motivation for the change:
Necessary for our different video upload flow with AWS Lambda

## How to test:
 - log in as johnny@example.com
 - go to the video upload page
 - upload a video
 - verify that the video uploads to the privacypal-input S3 bucket
 - verify that the processed video shows up in the privacypal-output S3 bucket after a minute or two (somewhat optional as this isn't the point of this PR but still this should be working)
